strawberryfields/parameters.py

```diff
@@ -257,6 +258,8 @@ def par_convert(args, prog):
     """
 
     def do_convert(a):
+        if isinstance(a, blackbird.RegRefTransform):
```

Comment:
Should this get tested as well?

Reply:
There is actually a test for this, but it was skipped. Just un-skipped it. It should pass when [Blackbird #46](https://github.com/XanaduAI/blackbird/pull/46) is in (at least it does for me locally).